app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss

```diff
@@ -0,0 +1,11 @@
+@import "govuk-frontend/components/accordion/accordion";
+
+.govuk-accordion--condensed {
+  .govuk-accordion__section-button {
+    @include govuk-font($size: 19, $weight: bold);
+  }
+
+  .govuk-accordion__section-summary {
+    @include govuk-font($size: 16);
```

Comment:
Hmm, it is intentional - but now not so sure that it's a good idea. 

I've changed this to not condense the accordion on smaller screens - since that's when the grid stacks columns on top of each other, and the default accordion works well in that context.